task1/calculator.html

```diff
@@ -0,0 +1,43 @@
+<html>
+  <head>
+    <title>
+      Calculator
+    </title>
+  </head>
+  <body>
+    <script language="JavaScript">
+      function addition(par1, par2)
+      {
+        return Number(par1) + Number(par2);
+      }
+      function minus(par1, par2)
+      {
+        return par1 - par2;
+      }
+      function multiplication(par1, par2)
+      {
+        return par1 * par2;
+      }
+      function division(par1, par2)
+      {
+        return par1 / par2;
+      }
+      function integerDivision(par1, par2)
+      {
+        return (par1 - par1 % par2) / par2;
```

Comment:
FYI `par1 - par1 % par2` is polyfill of Math.trunc
do not change it, it's just info